src/reducers/general.js

```diff
@@ -32,6 +43,10 @@ const general = (state = {
       return Object.assign({}, state, {
         pathname: action.pathname
       });
+    case types.CHANGE_LANGUAGE:
```

Comment:
same as comment above -- please comment out this code for the time being.

Reply:
I can't comment this out without breaking the language selector in the sidebar.